src/cli/bin.ts

```diff
@@ -45,6 +46,7 @@ yargs
     pivotInputPath?: string;
     debug: boolean;
     stack: boolean;
+    columnWidth?: string;
```

Comment:
I would prefer a more specific name. Otherwise it's ambiguous which column we are refering to.

```suggestion
    treeColumnWidth?: string;
```

Reply:
Done!